### PR TITLE
Ch/reference section responsiveness

### PR DESCRIPTION
**Pull request by IvyMurage**
What does this PR do?
It adds responsiveness to the reference section

Description of the task to be completed?

- Add responsiveness to the reference section for the mobile and medium-sized screens
- Change background-position

### CODE REVIEW

File: src/sections/sticky-yellow.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<div class="my-14 sm:my-28">
+<div class="my-14 sm:my-28 md:z-20">
   <img src="/src/assets/images/yellow-vector-2.svg" class="mx-auto block " />
```
Comment (tuyishimejohnson) [inline src/sections/sticky-yellow.html:2]: You can remove that space after block, in the class.

**Review by tuyishimejohnson — COMMENTED** [review]
You can find it on line two of sticky-yellow file.

**Review by tuyishimejohnson — CHANGES_REQUESTED** [review]
You can remove that space after block, in the class.

**Review by KevinRebakure — APPROVED** [review]
This looks good

**Review by tuyishimejohnson — APPROVED** [review]
It seems like everything looks good. Keep on!

**Pull request merged**